darts/utils/data/tabularization.py

```diff
@@ -0,0 +1,136 @@
+from typing import Optional, Sequence, Union
+
+import numpy as np
+import pandas as pd
+
+from darts.logging import raise_if
+from darts.timeseries import TimeSeries
+
+
+def create_lagged_data(
+    target_series: Union[TimeSeries, Sequence[TimeSeries]],
+    output_chunk_length: int,
+    past_covariates: Optional[Union[TimeSeries, Sequence[TimeSeries]]] = None,
+    future_covariates: Optional[Union[TimeSeries, Sequence[TimeSeries]]] = None,
+    lags_list: Optional[Sequence[int]] = None,
```

Comment:
could we remove the `_list` from the variable names here and below? Also adapt the type hints, if they are indeed lists.